Geometry/RPCGeometryBuilder/src/RPCGeometryBuilder.h

```diff
@@ -32,26 +31,25 @@ class RPCDetId;
 class RPCRoll;
 class MuonGeometryConstants;
 
-class RPCGeometryBuilderFromDDD {
+class RPCGeometryBuilder {
 public:
-  RPCGeometryBuilderFromDDD(bool comp11);
+  RPCGeometryBuilder(bool comp11);
```

Comment:
please, remove `comp11` - we are already using C++17 :-)

Reply:
done